cmd/kops/toolbox_template.go

```diff
@@ -166,14 +163,39 @@ func runToolBoxTemplate(f *util.Factory, out io.Writer, options *toolboxTemplate
 		if err != nil {
 			return fmt.Errorf("unable to render template: %s, error: %s", x, err)
 		}
-
+		if len(rendered) <= 0 {
+			continue
+		}
 		io.WriteString(writer, rendered)
```

Comment:
Probably better to call `writer.WriteString`, now that it is a bytes.Buffer (because we probably should have been checking for errors here previously, but we don't have to with bytes.Buffer, but we might as well call the method directly to trigger an error if we make this an io.Writer again!)  But not a big deal!